lms/services/oauth1.py

```diff
@@ -7,14 +14,12 @@ class OAuth1Service:
     def __init__(self, _context, request):
         self._request = request
 
-    def get_client(self):
+    def get_client(self) -> OAuth1:
```

Comment:
I don't go and add type hints everywhere but I can't resist when there's one of those `:rtype:` comments.